lib/main.dart

```diff
@@ -38,16 +39,14 @@ Future main([List<String> args = const []]) async {
     ).createSync(recursive: true);
   }
 
+  Directory directory = await getApplicationDocumentsDirectory();
+  FlutterNativeSplash.remove();
```

Comment:
Don't think this is needed, native splash is only supposed to go off after flutter has rendered its first frame which afaik does not happen till runApp anyway